proto/toit/api/hardware.proto

```diff
@@ -16,12 +16,14 @@ service HardwareService {
 message ClaimRequest {
   bytes hardware_id = 1;
   string init_device_name = 2;
+  bytes organization_id = 3;
```

Comment:
Add a comment that this is optional and default would be the current org for the user.